src/GEOM.jl

```diff
@@ -132,17 +182,72 @@ function sil_area_of_ellipsoid(a, b, c, θ)
     π * semax1  * semax2
 end
 
-function area_of_cylinder(r, l)
-    2 * π * r * l + 2 * π * r^2
+
+
+
+"""
+    Frog <: Shape
+
+An frog-shaped organism. Based on the leopard frog (Tracy 1976 Ecol. Monog.)
+"""
+struct Frog{M,D} <: Shape
+    mass::M
+    density::D
 end
 
+function geometry(shape::Frog, ::Naked)
+    volume = shape.mass / shape.density
+    length = volume^(1 / 3)
+    mass_g = Unitful.ustrip(Unitful.uconvert(u"g", shape.mass))
+    area = Unitful.uconvert(u"m^2", (12.79 * mass_g ^ 0.606)u"cm^2") # eq in Fig. 5
+    return Geometry(volume, length, nothing, area)
+end
+# function geometry(shape::Ellipsoid, fur::Fur)
+#     # Something different with fur
+#     # ...
+#     return Geometry(volume, length, (length1, length2, length3), area)
+# end
+
 """
-    sil_area_of_cylinder
+    sil_area_of_frog
 
-Calculates the silhouette (projected) area of a cylinder.
-Equation from Fig. 11.6 in Campbell, G. S., & Norman, J. M.
-(1998). Environmental Biophysics. Springer.
+Calculates the silhouette (projected) area of a leopard frog.
 """
-function sil_area_of_cylinder(r, l, θ)
-    2 * r * l * sin(θ) + π * r^2 * cos(θ)
+function sil_area_of_frog(area, θ)
+    pct = 1.38171e-6 * θ ^ 4 - 1.93335e-4 * θ ^ 3 + 4.75761e-3 * θ ^ 2 - 0.167912 * θ + 45.8228
+    sil_area = pct * area / 100
+end
+
+
+"""
+    Lizard <: Shape
+
+An lizard-shaped organism. (Porter and Tracy 1984)
+"""
+struct Lizard{M,D} <: Shape
+    mass::M
+    density::D
+end
+
+function geometry(shape::Lizard, ::Naked)
+    volume = shape.mass / shape.density
+    length = volume^(1 / 3)
+    mass_g = Unitful.ustrip(Unitful.uconvert(u"g", shape.mass))

```

Comment:
Whats the reason for using `ustrip`

Reply:
empirical functions like regressions tend to mess up the units - I've had to do this elsewhere in the code too

Reply:
Ahh of course. One option is to put he ustrip right where the empirical math happens, which is also a nice marker of where things are a bit less mechanistic

Reply:
Yep, that was the reason. I have moved ustrip to the actual regression equations. We could also do the uconvert in the same line, but it makes the line too long. Just an stylistic comment... what do you think?